docker-compose.yml

```diff
@@ -2,12 +2,14 @@ version: '3.7'
 
 services: # the different images that will be running as containers
   web: # service name
-    build: . # build the image for the web service from the dockerfile in parent directory.
+    build:
+      context: . # build the image for the web service from the dockerfile in parent directory.
+      target: development-stage
```

Comment:
the development stage is needed for running it locally, right?
Where are we setting the production stage as the target?

Reply:
also, would be great to have the reference added to Dockerfile: https://vuejs.org/v2/cookbook/dockerize-vuejs-app.html

Reply:
Docker-compose only works for local setup.

Production by default will run Dockerfile only.